tc_core/src/blueprint/for_config/server.rs

```diff
@@ -1,146 +1,76 @@
 use std::collections::{BTreeMap, BTreeSet};
+use std::net::IpAddr;
 
 use derive_setters::Setters;
+use hyper::HeaderMap;
 use serde::{Deserialize, Serialize};
 
-use crate::config::{is_default, KeyValues};
+use crate::blueprint::is_default;
 
-#[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq)]
-#[serde(rename_all = "camelCase")]
+#[derive(Clone, Debug, Setters)]
 pub struct Server {
```

Comment:
this is not what was suggested this should not move to bluprint it should be duplicated as did for server